frontend/lib/src/components/shared/BaseColorPicker/styled-components.ts

```diff
@@ -38,6 +38,12 @@ export const StyledChromePicker = styled.div(({ theme }) => ({
   div: {
     fontFamily: `${theme.genericFonts.bodyFont} !important`,
   },
+  label: {
+    fontSize: `${theme.fontSizes.sm} !important`,
+  },
+  input: {
+    fontSize: `${theme.fontSizes.sm} !important`,
+  },
```

Comment:
nit: is this something that cannot be done via `customChromePickerStyles`?

Reply:
Just investigated again. Seems like it's not possible and this is a known problem, see [this issue](https://github.com/casesandberg/react-color/issues/643). The `ChromeFields` component mentioned in there is the one [here](https://github.com/casesandberg/react-color/blob/master/src/components/chrome/Chrome.js#L134-L141) and you can see [here](https://github.com/casesandberg/react-color/blob/master/src/components/chrome/ChromeFields.js#L144-L162) that it applies default styles for `input` and `label` but doesn't offer a way to overwrite them.

Reply:
...and I assume this is also the reason why we set the `fontFamily` above and not through `customChromePickerStyles`.